test/boost/cql_auth_query_test.cc

```diff
@@ -32,6 +32,15 @@
 static const auto alice = std::string_view("alice");
 static const auto bob = std::string_view("bob");
 
+namespace seastar {
+
+std::ostream& boost_test_print_type(std::ostream& os, const std::unordered_set<seastar::sstring>& strings) {
+    fmt::print(os, "{}", strings);
+    return os;
+}
```

Comment:
just wanted to explain to other reviewers, if we see more needs for this helper, we will find it a better home, but before that, let's repeat it where it is used.